core/common/src/main/java/alluxio/conf/PropertyKey.java

```diff
@@ -3093,6 +3093,24 @@ public String toString() {
           .setConsistencyCheckLevel(ConsistencyCheckLevel.WARN)
           .setScope(Scope.WORKER)
           .build();
+  public static final PropertyKey WORKER_NETWORK_CACHE_MANAGER_QUEUE_MAX =
+      new Builder(Name.WORKER_NETWORK_CACHE_MANAGER_QUEUE_MAX)
+          .setAlias(Name.WORKER_NETWORK_ASYNC_CACHE_MANAGER_QUEUE_MAX)
```

Comment:
Remove `PropertyKey.WORKER_NETWORK_ASYNC_CACHE_MANAGER_QUEUE_MAX`  since it is now replaced by `PropertyKey.WORKER_NETWORK_CACHE_MANAGER_QUEUE_MAX` ?

Reply:
keep original property key since threadpool is only used for async cache